lib/policies_finder_publisher.rb

```diff
@@ -67,10 +67,31 @@ def details
         document_type: "policy"
       },
       show_summaries: false,
-      facets: [],
+      facets: facets,
     }
   end
 
+  def facets
+    [
+      {
+        key: "public_timestamp",
+        short_name: "Updated",
+        name: "Published",
```

Comment:
Now that I think about it, maybe that should be changed too. Not sure if there will be confusion with the difference. Not a blocker for merging anyways.